src/docdb/commands/connectNoSqlContainer.ts

```diff
@@ -7,16 +7,19 @@ import { IActionContext } from "@microsoft/vscode-azext-utils";
 import { KeyValueStore } from "../../KeyValueStore";
 import { ext } from "../../extensionVariables";
 import { NoSqlQueryConnection, noSqlQueryConnectionKey } from "../NoSqlCodeLensProvider";
+import { CosmosDBKeyCredential } from "../getCosmosClient";
 import { DocDBCollectionTreeItem } from "../tree/DocDBCollectionTreeItem";
 import { pickDocDBAccount } from "./pickDocDBAccount";
 
 export function setConnectedNoSqlContainer(node: DocDBCollectionTreeItem): void {
+    const root = node.root;
+    const keyCred: CosmosDBKeyCredential | undefined = root.credentials.filter((cred): cred is CosmosDBKeyCredential => cred.type === "key")[0];
```

Comment:
.at(0)

Reply:
The 'lib' in tsconfig is too old to support .at. Changing it results in multiple build error to surface. I'll skip using .at in this PR.